lib/bike_brigade_web/live/campaign_signup_live/show.html.heex

```diff
@@ -23,8 +23,8 @@
     <:col :let={task} label="Dropoff Neighbourhood">
       <%= Locations.neighborhood(task.dropoff_location) %>
     </:col>
-    <:col :let={task} label="Notes"><%= task.delivery_status_notes %>
```

Comment:
While swapping this, I noticed that for some reasons I added the delivery_status_notes into this html table. 🧹